components/eamxx/cmake/machine-files/ruby-intel.cmake

```diff
@@ -1,7 +1,8 @@
 include(${CMAKE_CURRENT_LIST_DIR}/ruby.cmake)
 set(CMAKE_EXE_LINKER_FLAGS "-L/usr/tce/packages/mkl/mkl-2022.1.0/lib/intel64/ -qmkl" CACHE STRING "" FORCE)
 set(PYTHON_EXECUTABLE "/usr/tce/packages/python/python-3.9.12/bin/python3" CACHE STRING "" FORCE)
+set(Python_EXECUTABLE "/usr/tce/packages/python/python-3.9.12/bin/python3" CACHE STRING "" FORCE)
```

Comment:
This line seems to be the same as the one above...

Reply:
It should but I've run into issues where it's looking for the lowercase Python and I'm not sure why

Reply:
Ah, I missed the upper/lower case difference.

That said, since you have `find_package(Python ..)` in `ml_correction/CMakeLists.txt`, this variable [should be set by CMake](https://cmake.org/cmake/help/latest/module/FindPython.html#result-variables).

Reply:
I tested it removing both of these python executable lines and it appeared to work.  So the latest branch version has these both gone.